src/Providers/Trustpilot/Request/Service/GetReviewsRequest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace PodPoint\Reviews\Providers\Trustpilot\Request\Service;
+
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Uri;
+use \PodPoint\Reviews\Request\BaseRequest;
+
+class GetReviewsRequest extends BaseRequest
+{
+    /**
+     * List of required fields.
+     *
+     * @return array
+     */
+    public function requiredFields(): array
+    {
+        return [];
+    }
+
+    /**
+     * Builds the request.
+     *
+     * @return Request
+     */
+    public function getRequest(): Request
+    {
+        $businessUnitId = $this->getOption('businessUnitId');
+
+        $method = 'GET';
+        $uri = new Uri("https://api.trustpilot.com/v1/private/business-units/{$businessUnitId}/reviews");
```

Comment:
I understand here it will be a bit less clean to create constant :thinking: 

Maybe for this we skip it, or you make the left hand side of the string a variable, and then add the /reviews bit in at the end manually :thinking: 

Another way would be with printf but that's maybe just overdoing it for something pretty inconsequential :thinking: What do you think?